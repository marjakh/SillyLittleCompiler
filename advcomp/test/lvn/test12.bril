@main {
  a: int = const 4;
  b: int = const 3;
  sum: int = add a b;
  c: int = const 5;
  prod: int = mul sum c;
  print prod;
}
