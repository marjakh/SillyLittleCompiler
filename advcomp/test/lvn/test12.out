@main {
  prod: int = const 35;
  print prod;
}
